tools/functional-tester/etcd-tester/lease_stresser.go

```diff
@@ -132,6 +123,18 @@ func (ls *leaseStresser) Stress() error {
 	if err := ls.setupOnce(); err != nil {
 		return err
 	}
+
+	conn, err := grpc.Dial(ls.endpoint, grpc.WithInsecure(), grpc.WithBackoffMaxDelay(1*time.Second))
```

Comment:
I decided to create and close connection for each test case. In that way, I need to create connection every time.